recipes/itsxpress/1.7.1/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "itsxpress" %}
+{% set version = "1.7.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "2e04503cb1c4261e75672dfb7c2999ce27fac539d8918ec5ebd53392216d2164" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - itsxpress=itsxpress.main:main
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True # [py27]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pip
+    - python
+    - biopython >=1.60
+    - hmmer >=3.1
+    - bbmap
+    - vsearch
+
+test:
+  imports:
+    - itsxpress
+  commands:
+    - itsxpress --help
+
+about:
+  home: http://github.com/usda-ars-gbru/itsxpress
+  license: 'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication'
+  license_family: 'PUBLIC-DOMAIN'
+  license_file: ''
+  summary: 'ITSxpress: Software to rapidly trim the Internally Transcribed Spacer (ITS) region from FASTQ files'
+
+
+extra:
+  recipe-maintainers: 'Adam R. Rivers'
```

Comment:
please add here your github username

Reply:
I wasn't quite sure the correct format but after looking at some other meta.yml examples I think this is correct:

```
 recipe-maintainers:
    - arivers
```